lib/graphql_metrics/extractor.rb

```diff
@@ -164,7 +175,7 @@ def extract_field(irep_node)
           type_name: irep_node.owner_type.name,
           field_name: irep_node.definition.name,
           deprecated: irep_node.definition.deprecation_reason.present?,
-          resolver_times: ctx_namespace.dig(TIMING_CACHE_KEY, irep_node.ast_node) || [],
+          resolver_times: ctx_namespace&.dig(TIMING_CACHE_KEY, irep_node.ast_node) || [],
```

Comment:
Likewise, `resolver_times` don't make sense to return if the extractor isn't run as instrumentation.